docs/source/acknowledge.rst

```diff
@@ -89,3 +89,11 @@ You can add the following badge to your **slides** or **README** files to acknow
         :target: https://signac.io
 
 Badges generated with `shields.io <https://shields.io>`_.
+
+Logos
+=====
+
+The **signac**, **signac-flow**, and **signac-dashboard** logos are licensed under the `Creative Commons Attribution 4.0 International (CC BY 4.0) license`_.
+This means you can use logos as long as you attribute the logos to the relevant **signac** project.
```

Comment:
Should we explicitly mention the badge here?

Reply:
explicit is better than implicit :)